tests/test_resource_tracker.py

```diff
@@ -157,6 +165,27 @@ def create_resource(rtype):
                                                 errno.ENOENT))
         assert re.search(expected, err) is not None
 
+    def test_resource_tracker_refcounting(self):
+        tmpfile = tempfile.NamedTemporaryFile(delete=False)
+        filename = tmpfile.name
+        tmpfile.close()
+        assert os.path.exists(filename)
+
+        from loky.backend.resource_tracker import _resource_tracker
+        _resource_tracker.register(filename, "file")
+        _resource_tracker.register(filename, "file")
+
+        _resource_tracker.unregister(filename, "file")
+        time.sleep(1)
+        # at this point, tmpfile was registered twice and unregistered
+        # once, so tmpfile should still exist
+        assert os.path.exists(filename)
```

Comment:
If we run this from a subprocess and capture stderr, we can also make sure the `resource_tracker` logs contains the patterns we want (registering/unregistering/unlinking/no error). @ogrisel WDYT?

Reply:
+1